packages/js/components/src/woo-product-section-item/woo-product-section-item.tsx

```diff
@@ -3,41 +3,50 @@
  */
 import React from 'react';
 import { Slot, Fill } from '@wordpress/components';
-import { createElement } from '@wordpress/element';
+import { createElement, Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { createOrderedChildren, sortFillsByOrder } from '../utils';
+import { ProductFillLocationType } from '../woo-product-tab-item';
 
 type WooProductSectionItemProps = {
 	id: string;
-	location: string;
+	tabs: ProductFillLocationType[];
 	pluginId: string;
-	order?: number;
 };
 
-type WooProductFieldSlotProps = {
-	location: string;
+type WooProductSectionSlotProps = {
+	tab: string;
 };
 
 export const WooProductSectionItem: React.FC< WooProductSectionItemProps > & {
-	Slot: React.FC< Slot.Props & WooProductFieldSlotProps >;
-} = ( { children, order = 20, location } ) => (
-	<Fill name={ `woocommerce_product_section_${ location }` }>
-		{ ( fillProps: Fill.Props ) => {
-			return createOrderedChildren< Fill.Props >(
-				children,
-				order,
-				fillProps
-			);
-		} }
-	</Fill>
-);
+	Slot: React.FC< Slot.Props & WooProductSectionSlotProps >;
+} = ( { children, tabs } ) => {
+	return (
+		<>
+			{ tabs.map( ( { name: tabName, order: tabOrder } ) => (
+				<Fill
+					name={ `woocommerce_product_section_${ tabName }` }
+					key={ tabName }
+				>
+					{ ( fillProps: Fill.Props ) => {
+						return createOrderedChildren< Fill.Props >(
+							children,
+							tabOrder || 20,
+							fillProps
```

Comment:
Exposing the `tabName` as part of the `fillProps` would be good to.

Reply:
Done in e6543756c253907b96119baedec6401cd0448076